src/components/UI/Display/Apps/AppList/AppsListPage.tsx

```diff
@@ -0,0 +1,108 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'styled-components';
+import Button from 'UI/Controls/Button';
+import App, { IAppProps } from 'UI/Display/Apps/AppList/App';
+import Facets, { IFacetOption } from 'UI/Inputs/Facets';
+
+import Toolbar from './Toolbar';
+
+const Wrapper = styled.div``;
+
+const ListAndFacets = styled.div`
+  display: flex;
+`;
+
+const List = styled.div`
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(270px, 1fr));
+  gap: 20px;
+  border-left: 1px solid ${({ theme }) => theme.colors.darkBlueLighter1};
+  width: 100%;
+  padding-left: 20px;
+`;
+
+const EmptyState = styled.div`
+  display: flex;
+  text-align: center;
+  width: 100%;
+  height: 300px;
+  border-left: 1px solid ${({ theme }) => theme.colors.darkBlueLighter1};
+  align-items: center;
+  justify-content: center;
+
+  h3 {
+    font-size: 18px;
+    margin-bottom: 0px;
+    font-weight: 400;
+  }
+`;
+
+export interface IAppsListPageProps {
+  matchCount: number;
+  onChangeFacets: (value: string, checked: boolean) => void;
+  facetOptions: IFacetOption[];
+  apps: IAppProps[];
```

Comment:
Why not use the actual app type? That's what will be coming here from the global state
https://github.com/giantswarm/happa/blob/389e0d4418cc418860968b4a219d8108b8f51742/src/%40types/appcatalogs.d.ts#L18

Reply:
I think it's to avoid that tight coupling between the state and the presentational layer. Makes it really obvious to see what properties of an App this component really cares about. 

Quicker to write a storybook about without having to create 'half representations' of an app?

I'm kinda justifying it after the fact, but it feels right to have this really clear boundary between presentation layer and business logic layer.. what do you think?